apps/leo_gateway/test/leo_nfs_integration_tests.sh

```diff
@@ -173,6 +195,8 @@ function check_dummy() {
     ls_validate_num_of_childs $MOUNT_DIR 504 &&
     eval $CMD_DD &&
     stat_validate_size $DST_50M_FILE4 128 &&
+    partial_write_test $TMP_50M_FILE $DST_50M_FILE3 524288 &&
+    partial_read_test $TMP_50M_FILE $DST_50M_FILE3 1572864 1048576 &&
```

Comment:
As this test is supposed to run on not only linux but other xunix we are supporting (freebsd/smartos,
We need to do either
1. Make drop_cache work on others (seems to be difficult but not impossible
2. Skip partial_(read|write)_test on others

For now, let's go with 2